### DISCUSSION TITLE
Question regarding wording for Vehicle type (OPT-155-LotResult)

### DISCUSSION THREAD

**Discussion by lesomue** [OP-TED/eForms-SDK#890] [Q&A] [answered]
Hello, 

while working on the new SDK 1.10.1, we came across an inconsistency regarding the wording of OPT-155-LotResult. I would like to explain our question with an example in eNotices2: 

The group Clean Vehicle Directive Information has the following hierarchy in a CAN notice:

1. CVD Contract type (BT-735)
1.1 Vehicle category (BT-723) 
1.1.1 Vehicle type (OPT-155) and Number of vehicles (OPT-156) 

![image](https://github.com/OP-TED/eForms-SDK/assets/117835335/3dc7a4fd-1d16-48f9-8049-af36518268f4)

That means that per vehicle category, it is possible to count each possible vehicle type once and the given numbers only apply to the named vehicle category within the named contract type. 
For each contract type (Passenger road transport services, Vehicle purchase, lease or rent, Other service contract), it is possible to choose each vehicle category (e. g. M1, M2, …) and for each vehicle type it is possible to enter the vehicle numbers for each of the vehicle types (The number of all vehicles falling within the scope of the CVD directive, The number of clean vehicles, The number of zero-emission heavy-duty vehicles).

However, in our understanding the wording of vehicle type „The number of all vehicles falling within the scope of the CVD directive“ suggests that the total number of all vehicles across all contracts and all vehicle categories should be specified here and not the total number of vehicles of this particular vehicle category, as specified via the hierarchy.

Could you please explain how OPT-155-LotResult „The number of all vehicles falling within the scope of the CVD directive“ should be understood here with regard to the discrepancy and let us know if it would be possible to change the wording into something less ambiguous? 

We have already placed this question via the TED Helpdesk but hope to find a quicker answer here. So many thanks in advance.

Best regards
Lea Müller
cosinex GmbH

**Comment by YvesJo** [comment, marked as answer]
Hi,
Following the Annex of the eForms amendment of 2022, all the statistics provided about vehicles (_BT-723 + BT-715, BT-725, BT-716_) are inside the Business Group _CVD information (BG-714)_ which is only about a single CVD Contract Type (BT-735 not being repeatable inside that Business Group). Statistics are therefore per CVD Contract Type and the Statistics for all the Contract Types may easily be inferred from the ones for the individual Contract Types.
The code labels are aligned with the description of the Business Term BT-715 in the Amending Regulation.
A label change could be considered and should be coordinated with DG GROW.
